src/Constants.ts

```diff
@@ -508,6 +522,16 @@ export class Constants {
     mainnet: 1,
     rinkeby: 4,
     ropsten: 3,
+    "arbitrum-mainnet": 42161,
```

Comment:
I think we need to get chainId's and the chain explorer code I have in my treeview merged together soon. Otherwise we will have duplicate/splitbrain code.

Reply:
Yes, you are :)